### PR TITLE
Feature/anymal c

### PR DESCRIPTION
**Pull request by Samu24**
PR adds anymal_c controller to the repo.

PR depends on: https://github.com/osrf/subt/pull/727

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,35 @@ This repository contains the software that can be used to control the ANYmal mod
...
+      	volume = {5},
+      	number = {47},
+      	elocation-id = {eabc5986},
+      	year = {2020},
+      	doi = {10.1126/scirobotics.abc5986},
+      	publisher = {Science Robotics},
+      	URL = {https://robotics.sciencemag.org/content/5/47/eabc5986},
+      	eprint = {https://robotics.sciencemag.org/content/5/47/eabc5986.full.pdf},
+      	journal = {Science Robotics}
+    }
```
Comment (marco-tranzatto) [inline README.md:39]: can you also add a link to https://robotics.sciencemag.org/content/5/47/eabc5986 ?

File: README.md [hunk 2]
```diff
@@ -43,16 +63,22 @@ cd ~/anymal_locomotion_ws/
...
 catkin config --extend ~/subt_ws/install/
 catkin config --cmake-args -DCMAKE_BUILD_TYPE=Release
-
-catkin build cerberus_anymal_b_control_1
 ```
 
-Launch the ANYmal locomotion controller:
+Build and launch the locomotion controller for ANYmal B:
 ```
+catkin build cerberus_anymal_b_control_1
```
Comment (timonh) [inline README.md:70]: Basic naming question: Why are we appending the 1? Is this still needed to discriminate something?

Comment (Samu24) [reply README.md:70]: This means controller config 1. For instance if you have a different policy you could name it _2.

Comment (Samu24) [reply README.md:70]: It follows the naming scheme with sensor_config_*

File: README.md (hunk 1 above)
Comment (Samu24) [reply README.md:39]: But it is already there. URL? Or what do you mean?

**Review by marco-tranzatto — APPROVED** [review]

**Pull request merged**